tests/pipelines/support/test_support_criteria_parser.py

```diff
@@ -0,0 +1,38 @@
+from budgetkey_data_pipelines.pipelines.supports.criteria.parser import parse_row
+from datapackage import DataPackage
+
+
+def test_support_criteria_parser():
+    dp = DataPackage({
+        "name": "support-criteria",
+        "resources": [{
+            "name": "criteria",
+            "dialect": {"delimiter": ",", "doubleQuote": True, "lineTerminator": "\r\n", "quoteChar": '"', "skipInitialSpace": False}, "encoding": "utf-8", "format": "csv",
+            "path": "tests/pipelines/support/criteria.csv",
+            "schema": {
+                "fields": [
+                    # the original support-criteria fields
+                    {"format": "%Y-%m-%d", "name": "date", "type": "date"},
+                    {"name": "title", "type": "string"},
+                    {"name": "paper_type", "type": "string"},
+                    {"name": "office", "type": "string"},
+                    {"format": "uri", "name": "pdf_url", "type": "string"},
+                    # the expected data from the parser
+                    {"name": "expected_purpose", "type": "string"}
+                ]
+            }
+        }]
+    })
+    i = 0
+    num_parsed = 0
+    num_expected_purposes = 0
+    for i, row in enumerate(dp.resources[0].iter()):
```

Comment:
should be `iter(keyed=True)`
(this was a breaking change in the `datapackage` library 🙁 )
will also fix the failing test I believe.